samplers/parser.go

```diff
@@ -356,6 +356,22 @@ func ParseMetric(packet []byte) (*UDPMetric, error) {
 		ret.Value = v
 	}
 
+	// Collect list of includes tags by prefix metric name.
+	// For example: in configuration file:
+	// 	tags_prefix_exclude_by_prefix_metric:
+	//   - metric_prefix: test.2xx
+	//     tags:
+	//       - foo
+	//
+	// if the metric is test OR test.2xx the the list of tags (foo) will add to excludeTags
+	var excludeTags []string
```

Comment:
do you really need another array to represent the excluded tags? we already have them set in the function arguments.
you can use a boolean toggle here to exclude if the prefix match.